test/unit/uk/gov/hmrc/apinotificationpull/controllers/SuccessfullHeaderValidator.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apinotificationpull.controllers
+
+import play.api.mvc.{ActionBuilder, Request, Result}
+import uk.gov.hmrc.apinotificationpull.validators.HeaderValidator
+
+import scala.concurrent.Future
+
```

Comment:
not sure why we need an validator implementation that always returns true? Can't we use actual header values to drive tests?

Reply:
The actual header values are driving the validator tests. This is a fake, injected into the controller from the controller unit tests. Seemed simpler/more expressive to create this fake manually rather than with mockito.